homeassistant/components/telekomunikace_2n/button.py

```diff
@@ -0,0 +1,87 @@
+"""2N Telekomunikace button platform."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any
+
+from py2n import Py2NDevice
+
+from homeassistant.components.button import (
+    ButtonDeviceClass,
+    ButtonEntity,
+    ButtonEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import Py2NDeviceCoordinator, Py2NDeviceEntity
+from .const import DOMAIN
+
+
+@dataclass
+class Py2NDeviceButtonRequiredKeysMixin:
+    """Class for 2N Telekomunikace entity required keys."""
+
+    press_action: Callable[[Py2NDevice], Any]
```

Comment:
Can we make the return type (`Any`) a little more specific?

Reply:
Yes we can! 😅